postBuild

```diff
@@ -1,2 +1 @@
-jupyter labextension install plotlywidget
```

Comment:
@ilyabo It looks like these are duplicative, and seems like they might've changed names? `plotlywidget` seems to be on an old version and hasn't been updated.
https://www.npmjs.com/package/plotlywidget
https://www.npmjs.com/package/jupyterlab-plotly

Reply:
The plotly example (`Crossfilter.ipynb`) still works on this branch after removing `plotlywidget`

Reply:
👍